DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -97,3 +97,66 @@
+* For Payara is needed to be replaced launcher.jar from Payara launcher.zip. For that:
+	- Payara server should be stopped
+	- go to https://youtrack.jetbrains.com/issue/IDEA-216528
+	- download launcher.jar file attached to the conversation(at least for windows PCs it will be downloaded as a ZIP file. So please rename it by writing JAR instead of ZIP)
+	- need to be deleted the cash from opt\domains\sormas\osgi-cache\felix (folder felix should be deleted as well)
+	- replace original launcher.jar file with downloaded (and renamed one) for next 2 paths: 1) opt\payara5\glassfish\launcher.jar and opt\payara5\glassfish\modules\launcher.jar
+* Only after fixing those issues, DEBUG should start well.
+
+## Issues which were found during installation process of the project
+**Note: Those are only points which were found by the developer who did this installation. They can be treated accordingly in order to speed up setting the development environment.
+* (1)If you set once, sormas_user password for postgres DB to `password1`, at next run of .\server-setup.sh you need to use the same password.
+If the password will be set to `password2`, it means that the file \opt\domains\sormas\config\domain.xml will have the `password2` set, but the DB will keep the old password for `sormas_user` created for postgres.
+
+* (2)Payara version. As by today (18.12.2020) the latest payara version is 5.2020.7
+From Alex V. I understood that the application cannot run with this version. Is needed to have maximum version of 5.2020.2 (this is the maximum version on which application does not have problems and can run)
+-> build script was not taking the Payara 5.2020.7 which was specified in it. I need to put the latest version in order to can run this script.
+Note. This was another major issue, seems I didn't know the application version numbering for Payara (last working stable build)
+In order to fix the issue, I just nee to keep next line: `PAYARA_VERSION=5.194` from the next if. (if was replaced)
+if [[ ${DEV_SYSTEM} != true ]]; then
+    PAYARA_VERSION=5.2020.2
+else
+    PAYARA_VERSION=5.194
+fi
+
+In comments, we do have there temporal workaround note. But nobody is saying that the application will not start on latest version
+=> Temporal workaround: Do not use newer version than 5.194 for developers to avoid redeployment issue, see https://github.com/hzi-braunschweig/SORMAS-Project/issues/2511
+
+PS. Alex V. who is using Payara version of 5.2020.2 - it attaching to the process in case of debugging and cannot start debug directly from the Intellij Idea.
+
+* (3) Passwords are really stored under the file: \opt\domains\sormas\config\domain.xml - but, at the moment when I put the right password, this file has been modified by another process which it putting back the old passwords. So, it is better to re-run ./server-setup.sh in order to put the right password for the DB.
```

Comment:
That's not really true, but might have been a side effect of start/deploy processes while editing. Changing domain.xml while the Payara domain is stopped does avoid such problems and makes sure that settings that can only be changed at domain start will be recognized.